koku/masu/processor/orchestrator.py

```diff
@@ -33,11 +34,14 @@
 from masu.processor.tasks import SUMMARIZE_REPORTS_QUEUE
 from masu.processor.tasks import SUMMARIZE_REPORTS_QUEUE_XL
 from masu.processor.worker_cache import WorkerCache
+from masu.util.aws.common import get_or_clear_daily_s3_by_date
 from masu.util.common import check_setup_complete
+from masu.util.common import get_path_prefix
 from subs.tasks import extract_subs_data_from_reports
 from subs.tasks import SUBS_EXTRACTION_QUEUE
 
 LOG = logging.getLogger(__name__)
+dh = DateHelper()
```

Comment:
I don't think this needs to be a global. An instance attribute seems more appropriate.

If this does end up being global, it should be all caps to indicate that.